microshift_running_apps/microshift-operators.adoc

```diff
@@ -10,9 +10,67 @@ You can use Operators with {microshift-short} to create applications that monito
\ No newline at end of file
+There are two ways to install Operators for your {microshift-short} clusters:
+
+* Operators can be installed directly with manifests. You can use the `kustomize` configuration management tool with {microshift-short} to deploy an application. Use the same steps to install Operators with manifests. See xref:../microshift_running_apps/microshift-applications.adoc#microshift-manifests-overview_applications-microshift[Using Kustomize manifests to deploy applications] for more information about manifests.
+
+* You can also install OLM-based Operators to a {microshift-short} cluster using Using Operator Lifecycle Manager (OLM).
+//TODO: see OSDOCS-8989 for summary and xref after PR is merged
+
+[id="microshift-olm"]
+== Using Operator Lifecyle Manager
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available, but you can take all the same actions with the CLI.
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+[id="microshift-file-based-olm-catalogs"]
+== File-based Operator catalogs
+
+To run {microshift-short} with your own curated catalogs and OLM, use the file-based catalog structure. For a complete explanation of the file-based structure, see the "Additional resources" section that immediately follows this one.
+
+include::modules/olm-fb-catalogs-example.adoc[leveloffset=+2]
+include::modules/olm-fb-catalogs-guidelines.adoc[leveloffset=+2]
+include::modules/olm-creating-fb-catalog-image.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/cli_tools/opm-cli[`opm` CLI reference]
+* link:https://https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/understanding-operators#olm-file-based-catalogs_olm-packaging-format[File-based catalogs]
+
+include::modules/olm-creating-catalog-from-index.adoc[leveloffset=+2]
+
+[id="microshift-using-olm-disconnected"]
+== Using OLM in disconnected clusters
+For {microshift-short} clusters that are installed on restricted networks, also known as _disconnected clusters_, Operator Lifecycle Manager (OLM) by default cannot access the Red Hat-provided OperatorHub sources hosted on remote registries because those remote sources require full internet connectivity.
```

Comment:
btw. `microshift-olm` does not ship any catalog, not even "Red Hat-provided OperatorHub sources" - should we rewrite this to not suggest it's shipped?